README.md

```diff
@@ -185,6 +185,34 @@ By default, this will run with the NodeJS versions that is configured in Foreman
 
 You can alter the behavior the same way as with the Ruby tests.
 
+## Smart Proxy plugin test
+
+Similar to the Foreman plugin test, this is aimed at Smart Proxy plugins.
+Conceptually it verifies certain Ruby versions are supported and using `test-gem.yml` to run the tests.
+
+```
+name: CI
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Tests
+    uses: theforeman/actions/.github/workflows/smart_proxy_plugin.yml@v0
+    with:
+      plugin_repository: $NAMESPACE/$REPOSITORY
```

Comment:
@evgeni I wonder if we can automatically derive this from the calling workflow. That way it should also work with forks. https://docs.github.com/en/actions/sharing-automations/reusing-workflows doesn't mention anything with related variables though.

Reading https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context I wonder if we could simply use `github.repository` to avoid needing to configure this.

Reply:
Eh? Just don't pass it. It will default to `github.repository`.

See https://github.com/theforeman/smart_proxy_pulp/blob/master/.github/workflows/main.yml

Reply:
I copied it from smart_proxy_openscap so I suppose this should be dropped there too. This is why good docs are important ;)

Reply:
you are here to fix my docs ;-)